public/docs/content/troubleshooting/new-package-projen-fail.md

```diff
@@ -0,0 +1,69 @@
+---
+title: How to add new project
+tags: projen
+packages: nx-monorepo, typescript-project
+---
+
+# I create a new package in my project, but running `npx projen` fails
+
+Make sure that when you create a new package you don’t set any interdependencies neither for the package nor of the package.
```

Comment:
This will be changing soon with the below PR. Once merged, you can create multiple projects with dependencies in one single synth thanks to @JeremyJonas  :)

https://github.com/aws/aws-prototyping-sdk/pull/195

Reply:
Yes, that's a great PR. shall we remove this item then?

Reply:
I think we can soon, but let's leave it for now.